cfme/tests/intelligence/reports/test_reports_schedules.py

```diff
@@ -34,19 +52,84 @@ def schedule(schedule_data, appliance):
         schedule.delete()
 
 
-@pytest.mark.sauce
-@pytest.mark.tier(3)
-@test_requirements.report
-def test_schedule_queue(schedule, appliance):
+@pytest.mark.parametrize("interval", TIMER)
+def test_schedule_queue(schedule, appliance, interval):
     """ To test scheduling of report using options: Once, Hourly, Daily, Weekly, Monthly
 
     Polarion:
         assignee: pvala
         casecomponent: Reporting
         caseimportance: high
         initialEstimate: 1/10h
+        tags: report
     """
 
     schedule.queue()
     view = schedule.create_view(ScheduleDetailsView)
-    view.flash.assert_message('The selected Schedule has been queued to run')
+    view.flash.assert_message("The selected Schedule has been queued to run")
+
+
+def test_reports_disable_enable_schedule(appliance, schedule):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: high
+        initialEstimate: 1/10h
+        tags: report
+    """
+    schedules = appliance.collections.schedules
+    schedules.disable_schedules(schedule)
+    assert not schedule.enabled
+    schedules.enable_schedules(schedule)
+    assert schedule.enabled
+
+
+@pytest.mark.ignore_stream("5.9")
+def test_report_schedules_invalid_email(appliance, schedule_data):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: medium
+        initialEstimate: 1/12h
+        tags: report
+    """
+    schedule_data["emails"] = (fauxfactory.gen_alpha(), fauxfactory.gen_alpha())
+    schedule_data["from_email"] = fauxfactory.gen_alpha()
+    with pytest.raises(AssertionError):
```

Comment:
These are the ```Emails``` then use ```fauxfactory.gen_email()```.

Reply:
I need to pass invalid email there, `fauxfactory.gen_email()` will generate a valid email, not a good idea to use it.